### PR TITLE
Adding monitor uptime widget doc

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?
Adds documentation for monitor uptime widget private beta.

### Motivation
PM request

### Preview link

* https://docs-staging.datadoghq.com/gus/monitor-uptime-widget/monitors/monitor_uptime_widget/

### CODE REVIEW

File: content/monitors/monitor_uptime_widget.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+kind: documentation
+description: "Track your monitor’s uptime via a widget in your Screenboard"
+private: true
+disable_toc: true
+---
+
+<div class="alert alert-warning">
+The Monitor Uptime Widget is currently in private beta. Interested in trying it out?  <a href="https://docs.google.com/forms/d/1EITy8IbRe484X-vfVyX8foU-_QXMyYKB6ciKXh3YUvU/edit">Request private beta access</a>.
+</div>
+
```
Comment (meghanelizabeth) [inline content/monitors/monitor_uptime_widget.md]: https://goo.gl/forms/KmWpQebCeeOdpPAq1

File: content/monitors/monitor_uptime_widget.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+Use the monitor Uptime widget to track your monitor’s uptime via a widget in your [Screenboards][1]. Monitor Uptime is defined as the amount of time a monitor was in an up state (OK) compared to down state (non-OK). Similar to the monitor status page visualization, the status is represented in bars as green (up) and red (down).
+
+## Widget configuration
+
+{{< img src="monitors/monitor_uptime_widget_configuration.png" alt="monitor uptime widget configuration" responsive="true" >}}
+
+To configure the widget from a [Screenboard][1].
+
+1. Select the monitor to display uptime from.
```
Comment (meghanelizabeth) [inline content/monitors/monitor_uptime_widget.md]: Select the monitor to display uptime for.

File: content/monitors/monitor_uptime_widget.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    Optionally - Select the monitor group ordering between the worst performing 5 groups, best performing 5 groups, or custom (select up to 5 custom groups). The total percentage when selected by group can be calculated: Based on the total uptime of the monitor or the average uptime for the selected groups.
```
Comment (meghanelizabeth) [inline content/monitors/monitor_uptime_widget.md]: Can this portion start on a new line? 
Optionally, you can view uptime by group for that monitor in three different ways: 
Worst performing 5 groups
Best performing 5 groups
Custom (select up to 5 groups) 
The total percentage when selected by group can be calculated based on the total of the monitor, or the average for the selected groups.

File: content/monitors/monitor_uptime_widget.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+2. If the widget is configured to report by groups, only one commune duration can be selected otherwise select up to 3 uptime durations for your widget between:
```
Comment (meghanelizabeth) [inline content/monitors/monitor_uptime_widget.md]: only one duration can be selected,

File: content/monitors/monitor_uptime_widget.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+## Widget behavior
+
+Find below a list of behavior description for the monitor uptime widget:
+
+* If the chosen monitor for the widget is configured to alert on warn or alert on no data, those states translates to down (red) inside the widget.
+
+* Downtimes and muted periods are disregarded: if your monitor triggers during a downtime that periods is removed from the calculated uptime.  
+
+While the widget is in private beta, Datadog engineering teams are still working through the final implementation for calculating the monitor uptime percentage. **Thus, until the end of the beta phase you mustn't rely on this percentage for reporting to your customers**. Should you see any discrepancies or unexpected behaviors, reach out to [Datadog support team][2]. 
+
```
Comment (meghanelizabeth) [inline content/monitors/monitor_uptime_widget.md]: Currently, if your monitor is configured to alert on warn or alert on no data, those states will translate to down (red) in the widget. Downtimes and muted periods will be disregarded, meaning, if your monitor triggers during an applied downtime that will period will be calculated as down.  

While the widget is in private beta, we are still working through the final implementation for calculating the percentage. Thus, you should not rely on this percentage for reporting to your customers for now. Should you see any discrepancies or unexpected behaviors, reach out to support@datadoghq.com.

Comment (meghanelizabeth) [reply content/monitors/monitor_uptime_widget.md]: Can we update this section

File: content/monitors/monitor_uptime_widget.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
+---
+title: Monitor Uptime Widget
+kind: documentation
+description: "Track your monitor’s uptime via a widget in your Screenboard"
+private: true
+disable_toc: true
+---
+
+<div class="alert alert-warning">
+The Monitor Uptime Widget is currently in private beta. Interested in trying it out? <a href="https://goo.gl/forms/KmWpQebCeeOdpPAq1">Request private beta access</a>.
```
Comment (phrawzty) [inline content/monitors/monitor_uptime_widget.md]: Don't need to say "currently" - the doc is always current, remember? 😉

File: content/monitors/monitor_uptime_widget.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+</div>
+
+{{< img src="monitors/monitor_uptime_widget.png" alt="monitor uptime widget" responsive="true" >}}
+
+Use the monitor Uptime widget to track your monitor’s uptime via a widget in your [Screenboards][1]. Monitor Uptime is defined as the amount of time a monitor was in an up state (OK) compared to down state (non-OK). Similar to the monitor status page visualization, the status is represented in bars as green (up) and red (down).
```
Comment (phrawzty) [inline content/monitors/monitor_uptime_widget.md]: "Monitor Uptime Widget" is camel-cased on [line 10](https://github.com/DataDog/documentation/pull/3235/files#diff-b36fb08e1ce89cb3a44564646f416a5fR10) but not here. I don't have strong opinions on capitalisation in general, but we should be consistent within the same document at least. 😄

File: content/monitors/monitor_uptime_widget.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+To configure the widget from a [Screenboard][1].
+
+1. Select the monitor to display uptime for.
+    
+    *Optionally* - Select the monitor group ordering between the worst performing 5 groups, best performing 5 groups, or custom (select up to 5 custom groups).
+
+    The total percentage when selected by group can be calculated based on the total uptime of the monitor or based on the average uptime for the selected groups.
+
+2. If the widget is configured to report by groups, only one duration can be selected otherwise select up to 3 uptime durations for your widget between:
```
Comment (phrawzty) [inline content/monitors/monitor_uptime_widget.md]: Add a semi-colon after "selected", or make it two sentences.

File: content/monitors/monitor_uptime_widget.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+    * 7 days 
+    * 30 days
+    * Month to date
+    * 90 days
+    * 365 days 
+    * Year to date
+
+    For 7 days the widget is restricted to 1 decimal place of accuracy. For 30 days and up the widget is restricted to 2 decimal places of accuracy. 
+
+3. Select the conditional formatting based on the uptime percentage, choose between: Red (down) / Green (up) / Yellow (warn).
```
Comment (phrawzty) [inline content/monitors/monitor_uptime_widget.md]: No need to say "choose between" if those are the only possible choices.

File: content/monitors/monitor_uptime_widget.md (hunk 7 above)
Comment (l0k0ms) [reply content/monitors/monitor_uptime_widget.md]: Uptime should be capitalised since it's the name of the widget, the rest should be all lower case

**Pull request merged**